consensus/src/main/java/org/apache/iotdb/consensus/iot/logdispatcher/IoTConsensusMemoryManagerMetrics.java

```diff
@@ -42,11 +42,29 @@ public void bindTo(AbstractMetricService metricService) {
         IoTConsensusMemoryManager::getMemorySizeInByte,
         Tag.NAME.toString(),
         "IoTConsensus");
+    metricService.createAutoGauge(
+        Metric.MEM.toString(),
+        MetricLevel.IMPORTANT,
+        iotConsensusMemoryManager,
+        IoTConsensusMemoryManager::getQueueMemorySizeInByte,
+        Tag.NAME.toString(),
+        "IoTConsensus_Queue");
```

Comment:
Remove `-` as you have used Camel-Case naming schema.

Reply:
ok